src/Microsoft.VisualStudio.Editors/SettingsDesigner/TypePickerDialog.vb

```diff
@@ -443,18 +444,18 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
                 PathSeparator = "."
                 Sorted = True
 
-                Dim assemblyImage As Bitmap = GetManifestBitmapTransparent("assembly.bmp", Color.FromArgb(255, 0, 255))
-                Dim namespaceImage As Bitmap = GetManifestBitmapTransparent("namespace.bmp", Color.FromArgb(255, 0, 255))
-                Dim objectImage As Bitmap = GetManifestBitmapTransparent("object.bmp", Color.FromArgb(255, 0, 255))
+                'Scale the imagelist for High DPI
+                Dim newSize = DpiAwareness.LogicalToDeviceSize(Handle, New Size(96, 96))
```

Comment:
We talked a little about this in person.  I am not certain this is the correct thing to do here.  Can you try this out with a monitor set to a different DPI setting and confirm that dragging from one to the other redraws correctly?

Reply:
Yes, it seemed to be drawn correctly across monitors